tests/test_server.py

```diff
@@ -1571,15 +1516,15 @@ async def test_multiple_identifiers():
 
     QGRAPH = query_graph_from_string(
         """
-        n0(( category biolink:ChemicalSubstance ))
-        n1(( category biolink:PhenotypicFeature ))
-        n1(( id UMLS:C0032961 ))
+        n0(( categories[] biolink:ChemicalSubstance ))
+        n1(( categories[] biolink:PhenotypicFeature ))
+        n1(( ids[] UMLS:C0032961 ))
         n0-- biolink:contraindicated_for -->n1
         """
     )
 
     # Create query
-    q = {"message" : {"query_graph" : QGRAPH}}
+    q = {"message" : {"query_graph" : QGRAPH}, "log_level": "DEBUG"}
```

Comment:
Is the addition of the log level needed for these tests?

Reply:
636530af0af7ea75682cd320b717f67d4937988b